### PR TITLE
EPMRPP-48754: Reporting. JS agent. Jasmine. Report static Attributes for suites / steps

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,45 @@ Example:
   "rerunOf": "f68f39f9-279c-4e8d-ac38-1216dffcc59c"
 ```
 
+### Report static attributes 
+To report attributes you need to do the following:
+1. Add PublicReportingAPI into your test file. 
+```javascript
+const PublicReportingAPI = require('reportportal-client/lib/publicReportingAPI');
```
Comment (DiscoElevator) [inline README.md]: I advise to re-export this functionality by the agent itself.
Reason: not every API may be implemented, so user will have set of not working functions. Also, I don't like an idea of exposing client itself. Let users work with the agent only.
```
const PublicReportingAPI = require('reportportal-agent-jasmine/...');
// ...
test('...', () => {
  PublicReportingAPI.addAttributes(...)
});
```

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   ],
   "main": "./lib/reportportal-agent",
   "dependencies": {
-    "reportportal-client": "^5.5.0"
+    "reportportal-client": "git://github.com/reportportal/client-javascript.git#master"
```
Comment (DiscoElevator) [inline package.json]: Please revert

File: lib/jasmine-reportportal-reporter.js [hunk 3]
```diff
@@ -34,17 +37,28 @@ class ReportportalReporter {
         return 'TEST';
     }
 
+    setAttributes(attributes) {
+        this.additionalCustomParams = { ...this.additionalCustomParams, ...attributes };
```
Comment (DiscoElevator) [inline lib/jasmine-reportportal-reporter.js]: Please use `Object.assign` here too to be able to support wider range of node versions.

File: README.md (hunk 1 above)
Comment (DiscoElevator) [reply README.md]: @AmsterGet what do you think?

File: package.json (hunk 2 above)
Comment (KatsiarynaTatarynovich) [reply package.json]: Reportportal-client V5.5.0 doesn't know about { Events }, so CI is failed. Should I skip this failure?

Comment (DiscoElevator) [reply package.json]: Let's skip this error for now.

Comment (KatsiarynaTatarynovich) [reply package.json]: If we want to skip this error we should skip all unit tests. I have the task with unit tests for jasmine. I can try to fix it as part of this task https://jiraeu.epam.com/browse/EPMRPP-46994

Comment (DiscoElevator) [reply package.json]: Can you just mock failing import for now?

Comment (KatsiarynaTatarynovich) [reply package.json]: I tried, but it didn't work. Ok, I will try better.

Comment (DiscoElevator) [reply package.json]: If won't work, just leave as is.

File: README.md (hunk 1 above)
Comment (AmsterGet) [reply README.md]: Definitely yes.
This was one of the points of such a decision.

Comment (AmsterGet) [reply README.md]: Decided to leave re-export approach

**Pull request merged**